internal/testing/listdeps.sh

```diff
@@ -33,4 +33,4 @@ function cleanup() {
 }
 trap cleanup EXIT
 
-sort "$tmpfile" | uniq
```

Comment:
Can you add a comment here explaining the added flags and why we're adding them?
I suppose it's something to the extent of different systems having different default behaviors for "sort", so we want to hardcode a very specific behavior

Reply:
Done.